src/server/models/log.js

```diff
@@ -0,0 +1,13 @@
+import thinky from './thinky'
+import { requiredString, timestamp } from './custom-types'
+
+const type = thinky.type
+
+const Log = thinky.createModel('log', type.object().schema({
```

Comment:
I think it's ok to create a table like this, but it should be optional to log to the database -- that should be a `process.env.LOG_TO_DB` or something, and managed from one of the `log()` implementations inside `./lib` directories (sadly, I think there are a few)